src/Libraries/System/Directory/Tree.idr

```diff
@@ -200,9 +200,17 @@ copyDir src target = runEitherT $ do
     MkEitherT $ createDir $ show target
     copyDirContents !(liftIO $ explore src) target
   where
+    -- Glue for the new and old definition of copyFile
+    partial
+    extract_error : (a : Type) -> a -> FileError
+    extract_error (Pair FileError _) (x, _) = x
+    extract_error FileError x = x
```

Comment:
This looks pretty dubious to me. I'd rather we use the usual approach of

1. put the new version in a `Libraries/` module
2. `import Libraries.Whatever as Lib`
3. use `Lib.copyFile'`
4. remove the `Lib` nonsense after the next release

Reply:
removed it :D